e2e/jest.test-setup.ts

```diff
@@ -212,7 +218,19 @@ const notRedirectRequest = (request: Request): boolean => {
 };
 
 const getResponseText = async (request: Request): Promise<string> => {
-  return (await request.response().buffer()).toString();
+  const REDIRECT_CODE_START = 299;
+  const REDIRECT_CODE_END = 400;
+  const NO_CONTENT_RESPONSE_CODE = 204;
+  const response = request.response();
+  // Log response if response it's not a redirect or no-content
+  const status = response && response.status() ? response.status() : null;
+  if (
+    status &&
+    !(status > REDIRECT_CODE_START && status < REDIRECT_CODE_END) &&
```

Comment:
I think it's a little easier to read values that are inclusive of the start and end. ex >=300 and <= 400.

Also, I think the redirect codes only range from 300-308.

Reply:
Updated.